<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindingWidget</class>
 <widget class="QWidget" name="FindingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="container" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#container {
	background: #EEF4F6;
	border-radius: 8px; 
}

#Header, 
#Body,
#FindingIDBox,
#AssigneeBox,
#AvatarBox {
	background: transparent;
}

#findingIDTextLabel, 
#finderLabel {
	font-family: Open Sans;
	font-weight: 600;
	font-size: 18px;
	line-height: 26px;
	letter-spacing: 0%;
	color: #1A1A1A;
	background: transparent;
}

#findingIDValueLabel {
	font-family: Open Sans;
	font-weight: 400;
	font-size: 14px;
	line-height: 22px;
	letter-spacing: 0%;
	color: #4D4D4D;
	background: transparent;
}

#assigneeTextLabel,
#dueDateLabel, 
#descriptionLabel {
	font-family: Open Sans;
	font-weight: 500;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
	color: #1A1A1A;
	background: transparent;
}

#assigneeValueLabel {
	font-family: Open Sans;
	font-weight: 500;
	font-size: 16px;
	line-height: 22px;
	letter-spacing: 0%;
	color: #4D4D4D;
	background: transparent;
}

#assigneeButton,
#avatar {
	background: transparent;
	border: none;
}

#verifyButton {
	background: #22084B;
	border: none;
	border-radius: 8px;
	padding: 7px 18px;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 16px;
	line-height: 22px;
	letter-spacing: 0%;
	color: white;
	text-decoration: none;
}

#commentButton {
	background: transparent;
	border: none;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
	color: #1A1A1A;
	text-decoration: underline;
}
	</string>
     </property>
     <layout class="QVBoxLayout" name="containerLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QWidget" name="Header" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>36</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>18</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="FindingIDBox" native="true">
           <layout class="QHBoxLayout" name="findingIDLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="findingIDTextLabel">
              <property name="text">
               <string>Finding ID: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="findingIDValueLabel">
              <property name="text">
               <string>FWYGH324</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="AssigneeBox" native="true">
           <layout class="QHBoxLayout" name="assigneeLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="assigneeTextLabel">
              <property name="text">
               <string>Assignee:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="assigneeValueLabel">
              <property name="text">
               <string>Travis Woth</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="assigneeButton">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../../Resources/Resources.qrc">
             <normaloff>:/images/Common/TableView/edit.svg</normaloff>:/images/Common/TableView/edit.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="verifyButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>Verified</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="Body" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>128</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="AvatarBox" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="avatarLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="avatar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../../../Resources/Resources.qrc">
                <normaloff>:/images/Reviews/default_avatar.svg</normaloff>:/images/Reviews/default_avatar.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="finderLabel">
              <property name="text">
               <string>James Tim H</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>744</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="dueDateLabel">
              <property name="text">
               <string>March 25, 2025</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="descriptionLabel">
           <property name="text">
            <string>Lorem ipsum dolor sit amet consectetur. Amet lacinia ut dignissim facilisis mattis.. Neque varius eget dolor nisl. Nulla sagittis pellentesque turpis faucibus tempor.. Congue ut sit adipiscing amet faucibus tempor.. </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="commentButton">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>1 Comment</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
